include/range/v3/view_interface.hpp

```diff
@@ -297,6 +297,19 @@ namespace ranges
             {
                 return Slice{}(detail::move(derived()), offs.from, offs.to);
             }
+            /// Returns a reference to the element at specified location pos, with bounds checking.
+            template<typename D = Derived,
+                CONCEPT_REQUIRES_(Same<D, Derived>() && RandomAccessRange<D>())>
+            auto at( range_difference_type_t<D> n) ->
+                decltype(std::declval<D &>().begin()[n])
+            {
+                using size_type = decltype( derived().size() );
+                if ( (n < 0) or
+                     (size_type(n) >= derived().size()) ){
```

Comment:
Ditto `or` and `(n < 0)` and `( foo )`. I would use `ranges::size(derived())` instead of `derived().size()` since that's the expression that `SizedRange` requires to be valid. (It's possible that the default member implementations of `size` don't work and some user implements non-member `size` for their view.)